README.md

```diff
@@ -1,2 +1,49 @@
 # Base-Android-Template
-This project is a structured template for an Android app that can be used as a starting point for creating your own apps
+This project is a structured template for an Android app that can be used as a starting point to create your own apps.
+
+
+## Introduction
+If you want to start an Android project with a <b><i>clean architecture, reactive programming integration with Coroutines, Room database usage, Retrofit integration, Locale changer and other basic concepts used in any Android application</i></b>, this code is what you need.
+
+To prove the mentioned concepts, the full-screen <b>Base-Android-Template</b> application contains a BotttomNavigationView with two sections:
+- <b>Github Users</b> is an interactive screen animated using the MotionLayout component. This screen displays a list of Github users fetched from the local Room database. If database does not contain users, the list will be retrieved from the server and then saved in Room. 
+- <b>Settings</b> is the screen where you can switch from Romanian and English languages. Here you can understand how simple and intuitive the Android Locale configurations can be changed.
```

Comment:
"Here you can understand how simple and intuitive the Android Locale configurations can be changed"
Here you can understand how simple and intuitive the Android Locale configurations can be.